src/cppcache/src/LRUList.hpp

```diff
@@ -20,32 +20,32 @@
  * limitations under the License.
  */
 
+#include <atomic>
+
 #include <geode/geode_globals.hpp>
 #include <geode/SharedPtr.hpp>
-#include "SpinLock.hpp"
+
+#include "util/concurrent/spinlock_mutex.hpp"
 
 namespace apache {
 namespace geode {
 namespace client {
+
 // Bit mask for recently used
-#define RECENTLY_USED_BITS 1ul
+#define RECENTLY_USED_BITS 1u
 // Bit mask for evicted
-#define EVICTED_BITS 2ul
+#define EVICTED_BITS 2u
```

Comment:
They are macros, but they could be converted to `const` in anonymous namespace or even in this namespace or `constexpr` on the class itself. I will look at what that looks like and maybe update it.